setup.py

```diff
@@ -33,10 +33,12 @@
 install_requires = [
     'fmf>=0.9.2',
     'click',
+    'requests'
 ]
 extras_require = {
     'docs': ['sphinx', 'sphinx_rtd_theme'],
     'tests': ['pytest', 'python-coveralls', 'mock'],
+    'provision': ['testcloud']
```

Comment:
pip fails to install `testcloud` unless `libvirt-devel` is present on the system. How shall we handle this? Add to documentation?

Reply:
Yes, I will add it to documentation